ui/js/blocks/src/blocks/components/BlockPreview.js

```diff
@@ -19,6 +19,7 @@ import {
 } from '@wordpress/date';
 
 import ServerSideRender from '@wordpress/server-side-render';
+import { PodsServerSideRender } from './PodsServerSideRender';
```

Comment:
```suggestion
import PodsServerSideRender from './PodsServerSideRender';
```
You can import this without destructuring since it's the default export.